### ISSUE TITLE
regarding incompatible torch with coda requirement

### ISSUE THREAD

**Issue by Vzenun** [sen-mao/StyleDiffusion#8]
I encountered the following error:
ERROR: Could not find a version that satisfies the requirement torch==1.12.1+cu113 (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.4.0, 2.4.1, 2.5.0)
ERROR: No matching distribution found for torch==1.12.1+cu113

Unfortunately, +cu113 is not provided for PyTorch 1.12.1.

so how do i run this

**Comment by sen-mao** [comment]
Perhaps you could try `pip install torch==1.11.0+cu113 torchvision==0.12.0+cu113 torchaudio==0.11.0 --extra-index-url https://download.pytorch.org/whl/cu113`

**Comment by Vzenun** [comment]
still not working ERROR: Could not find a version that satisfies the requirement torch==1.12.1+cu113 (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.4.0, 2.4.1, 2.5.0, 2.5.1)
ERROR: No matching distribution found for torch==1.12.1+cu113

**Comment by sen-mao** [comment]
It works well for me
![image](https://github.com/user-attachments/assets/96a3f38c-fc1a-49fe-9263-61af606082d9)

**Comment by Vzenun** [comment]
Yes but while wxecuting requirements and downloading it is showing this
error.
